industrial_ci/rosdep-install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -x
+#!/bin/bash -ex
```

Comment:
That's rather confusing as "-ex" might look like "exit".
I will prepare another commit :)